www/docs/intro.md

```diff
@@ -376,6 +376,14 @@ Devnet offers [empty block creation](https://shard-labs.github.io/starknet-devne
 const emptyBlock = await starknet.devnet.createBlock();
 ```
 
+#### Mint tokens to an account
```

Comment:
It's ok that minting is explained here, but there is also a paragraph in this file called `Fund account`, and I believe it should contain a link to this new pagraph `Mint tokens to an account`.

Reply:
Aye, it should!